<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>caculatorDialog</class>
 <widget class="QDialog" name="caculatorDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>642</width>
    <height>302</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Rainy Caculator</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>100</y>
     <width>419</width>
     <height>201</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="5" column="3">
     <widget class="QPushButton" name="del_pbt">
      <property name="text">
       <string>Del</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QPushButton" name="dot_pbt">
      <property name="text">
       <string>.</string>
      </property>
     </widget>
    </item>
    <item row="5" column="2">
     <widget class="QPushButton" name="v0_pbt">
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="QPushButton" name="v2_pbt">
      <property name="text">
       <string>2</string>
      </property>
     </widget>
    </item>
    <item row="1" column="4">
     <widget class="QPushButton" name="dev_pbt">
      <property name="text">
       <string>/</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QPushButton" name="v5_pbt">
      <property name="text">
       <string>5</string>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QPushButton" name="v9_pbt">
      <property name="text">
       <string>9</string>
      </property>
     </widget>
    </item>
    <item row="4" column="3">
     <widget class="QPushButton" name="v3_pbt">
      <property name="text">
       <string>3</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QPushButton" name="v4_pbt">
      <property name="text">
       <string>4</string>
      </property>
     </widget>
    </item>
    <item row="3" column="4">
     <widget class="QPushButton" name="sub_pbt">
      <property name="text">
       <string>-</string>
      </property>
     </widget>
    </item>
    <item row="3" column="3">
     <widget class="QPushButton" name="v6_pbt">
      <property name="text">
       <string>6</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QPushButton" name="v7_pbt">
      <property name="text">
       <string>7</string>
      </property>
     </widget>
    </item>
    <item row="4" column="4">
     <widget class="QPushButton" name="add_pbt">
      <property name="text">
       <string>+</string>
      </property>
     </widget>
    </item>
    <item row="2" column="4">
     <widget class="QPushButton" name="mul_pbt">
      <property name="text">
       <string>*</string>
      </property>
     </widget>
    </item>
    <item row="5" column="4">
     <widget class="QPushButton" name="equal_pbt">
      <property name="text">
       <string>=</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QPushButton" name="v1_pbt">
      <property name="text">
       <string>1</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QPushButton" name="v8_pbt">
      <property name="text">
       <string>8</string>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QPushButton" name="right_pbt">
      <property name="text">
       <string>)</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QPushButton" name="left_pbt">
      <property name="text">
       <string>(</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="clear_pbt">
      <property name="font">
       <font>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="text">
       <string>C</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLineEdit" name="exp_edt">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>421</width>
     <height>101</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string/>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="conclude_pedt">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>0</y>
     <width>221</width>
     <height>301</height>
    </rect>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::TextEditorInteraction</set>
   </property>
  </widget>
  <widget class="QPushButton" name="clearTextPlainEdit_pbt">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>280</y>
     <width>31</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>C</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
